source/jquery.flot.touchNavigate.js

```diff
@@ -7,6 +7,9 @@
         pan: {
             enableTouch: false,
             touchMode: 'manual'
+        },
+        recenter: {
+            enableTouch: false
```

Comment:
Like I suggest in the above comment, this option could move into the pan set of options and be called 'recenterOnDoubleTap' (or, again, 'allowRecenter'), and default to true. Again, recentering should only work when pan.enableTouch is set to true.

Reply:
Agree and fixed.